### DISCUSSION TITLE
如何删除下载队列中的记录

### DISCUSSION THREAD

**Discussion by youtehub** [Richasy/Bili.Uwp#1528] [展示]
如果想重复下载以前的视频，会由于在下载队列中，存在记录无法重复下载。下面两张图是已将删除后的对比照片。
![image](https://user-images.githubusercontent.com/18629921/195635939-a4741a08-1b72-4944-99d0-3b518a9196d7.png)
![image](https://user-images.githubusercontent.com/18629921/195635993-7971fa10-0270-4279-8013-4c12f26fd9b9.png)
![image](https://user-images.githubusercontent.com/18629921/195636605-e77be5a4-2699-452a-93d5-a93fc2af6e0c.png)

**建议**：下载的视频删除之后，顺带将下载队列中的记录也一起删除，不然无法重新下载相同的视频。
现在想重复下载视频的话，就是删除哔哩UWP后，重新安装才能操作

**Comment by Richasy** [comment]
这个不是Bili，这是官方应用
